scripts/build_info.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+HEAD=$(git rev-parse --verify HEAD --short)
+DATE=$(date +%Y-%m-%d)
+TIME=$(date +%H:%M:%S)
+COMMITS=$(git log --no-merges --pretty=format:'%h|%s|%aN' -n 10)
+DEV_INFO=${DEV_INFO:-false}
```

Comment:
I guess this is the environment variable that does the magic? Is not explained in the commit message.

Reply:
I've changed the environment variable needed to `SHOW_BUILD_INFO` and add it to the commit message to it's more clear.